observability/distr_tracing/distr-tracing-rn.adoc

```diff
@@ -12,9 +12,178 @@ You can use the {TempoName} xref:../otel/otel-forwarding-telemetry-data.adoc#ote
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=2312579[Cross-site Scripting (XSS) in `serialize-javascript`]
+
+[id="distr-tracing_3-4_tempo-release-notes_{context}"]
+=== {TempoName}
+
+The {TempoName} 3.4 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/tempo-operator-bundle/642c3e0eacf1b5bdbba7654a/history[{TempoOperator} 0.14.1].
+
+[NOTE]
+====
+The {TempoName} 3.4 is based on the open source link:https://grafana.com/oss/tempo/[Grafana Tempo] 2.6.1.
+====
+
+[id="distr-tracing_3-4_tempo-release-notes_technology-preview-features_{context}"]
+==== Technology Preview features
+
+This update introduces the following Technology Preview feature:
+
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+
+:FeatureName: Each of these features
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[id="distr-tracing_3-4_tempo-release-notes_new-features-and-enhancements_{context}"]
+==== New features and enhancements
+
+This update introduces the following enhancements:
+
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+* ???.
+
+[id="distr-tracing_3-4_tempo-release-notes_deprecated-functionality_{context}"]
+==== Deprecated functionality
+
+* In the {TempoName} 3.4, ???.
+* In the {TempoName} 3.4, ???.
+* In the {TempoName} 3.4, ???.
+
+[id="distr-tracing_3-4_tempo-release-notes_removal-notice_{context}"]
+==== Removal notice
+
+In the {TempoName} 3.4, the FEATURE has been removed. Bug fixes and support are provided only through the end of the 3.? lifecycle. As an alternative to the FEATURE for USE CASE, you can use the ALTERNATIVE instead.
+
+[id="distr-tracing_3-4_tempo-release-notes_bug-fixes_{context}"]
+==== Bug fixes
+
+This update introduces the following bug fixes:
+
+* Before this update, the {TempoShortName} failed on the {ibm-z-title} (`s390x`) architecture. With this update, the {TempoShortName} is available for the {ibm-z-title} (`s390x`) architecture. (link:https://issues.redhat.com/browse/TRACING-3545[TRACING-3545])
+Before this update, the {TempoShortName} failed on clusters with non-private networks. With this update, customers can deploy the {TempoShortName} on clusters with non-private networks. (link:https://issues.redhat.com/browse/TRACING-4507[TRACING-4507])
+* Before this update, the Jaeger UI could fail due to reaching a trace quantity limit, resulting in the *504 Gateway Time-out* error in the `tempo-query` logs. After this update, the issue is resolved by introducing two optional fields in the `tempostack` or `tempomonolithic` custom resource:
++
+** New `spec.timeout` field for configuring the timeout.
+** New `spec.template.queryFrontend.jaegerQuery.findTracesConcurrentRequests` field for improving the query performance of the Jaeger UI.
++
+[TIP]
+====
+One querier can handle up to 20 concurrent queries by default. Increasing the number of concurrent queries further is achieved by scaling up the querier instances.
+====
```

Comment:
@pavolloffay, could you please review this? I rewrote your draft in https://github.com/openshift/openshift-docs/pull/83770/files#r1832705037, but I have little confidence in my wording. Perhaps some things need to be corrected. WDYT?